.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for this project
```

Comment:
Feedback from @drewwills 
We want to avoid the issue tracker become a giant wishlist without any commitment toward completing tasks.
We should include language to further encourage participation, particularly around pull requests.

Reply:
@jimhelwig Would Apereo Farm be a good resource to refer people to if they are interested and motivated to have a feature, but don't have the technical resources available?